src/octopeer-github/test/resources/README.md

```diff
@@ -0,0 +1,4 @@
+!!! IMPOTANT! READ BEFORE EDITING THE FILES IN THIS FOLDER !!!
+
+The files in this folder are used to test the set of ElementSelectionBehaviour classes. Changing any of these files will most likely result in a huge amount of failing tests.
+Make sure you contact the responsible person (Mathias) and are cleared to edit the html files before doing so.
```

Comment:
> Make sure you [...] are cleared to edit the html files [...]
Eh... cleared? XD
Also, when we will be done with this project, it will probably no longer hold that you should be contacted. Maybe something like "only edit on your own risk" or "take a look at these test cases before editing"?